### DISCUSSION TITLE
Air drop 10.000 usdt Fake ?

### DISCUSSION THREAD

**Discussion by Azot67** [blockscout/blockscout#10178] [Q&A]
Hello, i juste receveid one Qr code nft gift 🎁 and i dont know if this IS Fake or something, the gift was by this website so i Ask your advice to get this please, IAM french 🇫🇷 thinks you

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Comment by Azot67** [comment]
Thx dude

Le lun. 3 juin 2024 à 13:53, Maxim Filonov ***@***.***> a
écrit :
